lib/utils/attributes_validator/index.js

```diff
@@ -32,6 +33,11 @@ module.exports = {
    */
   validate: function(attributes) {
     if (typeof attributes === 'object' && !Array.isArray(attributes) && attributes !== null) {
+      _.forEach(attributes, function(value, key) {
```

Comment:
Since `attributes` is an Object, it is more predictable to loop using `forOwn`. Include forOwn directly from lodash as we do in this file to avoid pulling in the entire lodash library: https://github.com/optimizely/node-sdk/blob/devel/lib/core/event_builder/index.js#L17